docker-controller-bot.py

```diff
@@ -922,6 +973,27 @@ def button_controller(call):
 	elif comando == "deleteSchedule":
 		delete_line_from_file(FULL_SCHEDULE_PATH, f'{schedule} {action} {containerName}')
 		send_message(message=get_text("deleted_schedule", f'{schedule} {action} {containerName}'))
+	# System Prune Container
+	elif comando == "prune":
+		if action == "confirmPruneContainers":
+			confirm_prune_containers()
+		elif action == "prune_containers":
```

Comment:
I would use camelCase for all actions (1 character less) because in callback_data we are limited to 64 characters.